apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -314,22 +312,6 @@ const useDrawerStyles = createStyles((theme: MantineTheme) => {
   };
 });
 
-export interface IIntegratedProvider {
```

Comment:
I think it was @p-fernandez that noticed this in one of my previous PRs and now I am handling this technical debt.